WordPressAuthenticator/GoogleSignIn/Data+SHA256.swift

```diff
@@ -0,0 +1,35 @@
+import CommonCrypto
+
+extension Data {
+
+    func sha256Hashed() -> Data {
+        Data(sha256Hash())
+    }
+
+    func sha256Hashed() -> String {
+        sha256Hash().map { String(format: "%02x", $0) }.joined(separator: "")
+    }
```

Comment:
I like the overload here, but it comes with the downside that some usages might require the type to be explicitly specified, `let x: String = data.sha256Hashed()` vs `let x = data.sha256Hashed()`.

The reason I needed both `Data` and `String` was for ease of testing.

Reply:
I've found the compiler generally can figure it out these days without too much `couldn't resolve this in reasonable time` – I like this overload pattern too!